apigenerator/APIGenerator.java

```diff
@@ -20,31 +20,39 @@
 	final Set<Class> m_VisitedClasses = new TreeSet<Class>(CLASSNAME_COMPARATOR);
 	final Set<Class> m_DependencyChain = new LinkedHashSet<Class>();
 
+	static final String k_UsageMessage = "Usage: APIGenerator <dst> [-s] <jarfile[;jarfile;...]> <regex...>\n";
+
 	public static void main(String[] argsArray) throws Exception
 	{
 		LinkedList<String> args = new LinkedList<String>(Arrays.asList(argsArray));
 		if (args.size() < 2)
 		{
-			System.err.format("Usage: APIGenerator <dst> <jarfile[;jarfile;...]> <regex...>\n");
+			System.err.format(k_UsageMessage);
 			System.exit(1);
 		}
 		String dst = args.pollFirst();
 		if (!new File(dst).isDirectory())
 		{
-			System.err.format("Usage: APIGenerator <dst> <jarfile[;jarfile;...]> <regex...>\n");
+			System.err.format(k_UsageMessage);
 			System.err.format("%s: is not a directory.\n", dst);
 			System.exit(1);
 		}
-
-		String jarList = args.pollFirst();
+		String nextArgument = args.pollFirst();
+		boolean useSystemClasses = false;
+		if (nextArgument.equals("-s"))
```

Comment:
Better to use opposite for the null-safety:
if ("-s".equals(nextArgument))

Reply:
That's a great suggestion, thanks. Changed in https://github.com/Unity-Technologies/android-jni-bridge/pull/41/commits/ff40b49e4ecef4d81045459090bd51f10888a735